docs/reference/changelog.md

```diff
@@ -19,27 +19,41 @@
 * [#355](https://github.com/open-contracting/infrastructure/pull/355) - use correct normative and non-normative keywords in schema descriptions.
 * [#361](https://github.com/open-contracting/infrastructure/pull/361) - clarify project budget description.
 * [#365](https://github.com/open-contracting/infrastructure/pull/365) [#386](https://github.com/open-contracting/infrastructure/pull/386) - add description field to budget.
-* [#367](https://github.com/open-contracting/infrastructure/pull/367) - add approval date to budget breakdown.
 * [#368](https://github.com/open-contracting/infrastructure/pull/368) - clarify contracting processes id description.
-* [#379](https://github.com/open-contracting/infrastructure/pull/379) - add `contractingProcesses.summary.tender.datePublished`.
 * [#378](https://github.com/open-contracting/infrastructure/pull/378) - clarify semantics of `additionalClassifications`.
-* [#390](https://github.com/open-contracting/infrastructure/pull/390) - add `identifiers`.
-* [#426](https://github.com/open-contracting/infrastructure/pull/426) - add fields:
-  * `.transactions`
-  * `.milestones`
-  * `ContractingProcesses.milestones`
-* [#427](https://github.com/open-contracting/infrastructure/pull/427) - add `Organization.beneficialOwners`
+* Add fields:
+  * [#367](https://github.com/open-contracting/infrastructure/pull/367) - `BudgetBreakdown.approvalDate`
+  * [#379](https://github.com/open-contracting/infrastructure/pull/379) - `.contractingProcesses.summary.tender.datePublished`
+  * [#390](https://github.com/open-contracting/infrastructure/pull/390) - `.identifiers`
+  * [#426](https://github.com/open-contracting/infrastructure/pull/426):
+    * `.transactions`
+    * `.milestones`
+    * `ContractingProcesses.milestones`
+  * [#427](https://github.com/open-contracting/infrastructure/pull/427) - `Organization.beneficialOwners`
```

Comment:
Use schema order and avoid references to definitions, to make it easier on users. (It's only necessary to refer to definitions when that definition is reused.)

```suggestion
  * [#390](https://github.com/open-contracting/infrastructure/pull/390) - `identifiers`
  * [#427](https://github.com/open-contracting/infrastructure/pull/427) - `parties.beneficialOwners`
  * [#426](https://github.com/open-contracting/infrastructure/pull/426) - `transactions`
  * [#426](https://github.com/open-contracting/infrastructure/pull/426) - `milestones`
  * [#379](https://github.com/open-contracting/infrastructure/pull/379) - `contractingProcesses.summary.tender.datePublished`
  * [#426](https://github.com/open-contracting/infrastructure/pull/426) - `contractingProcesses.summary.milestones`
```